### ISSUE TITLE
[ BUG]  Legder Nano S can't effective authorization

### ISSUE THREAD

**Issue by wjdfx** [nearprotocol/stakewars#541]
**Describe the bug or test**
I posted it on StackOverflow and it didn't respond, maybe it's not the right place.I discovered this problem when I was writing the ledger tutorial.

**How to Reproduce**
I use ledger nano S for https://wallet.betanet.near.org/, and the ledger nano S successful configuration. after I delete all key from my account and just keep 1 Full Access Key(create by ledger nano S).

Now I can't add other fullaccess key. Whatever method is used. When I did: near login --useLedgerKey. the web verify page will show:

An error occurred while approving this action. Please try again!

When I did add Seed Phrase enable, on the [Enter the following word from your recovery phrase to complete the setup process.] page, When I confirm it in the Ledger it will show:

_An error occurred while setting up seed phrase recovery. Please try again!_

Actually , my ledger still will show confirm info:

_DANGER: This gives full access to a device other than Ledger_

when I make a transaction push √ button. it still show the error:

_An error occurred while approving this action. Please try again!_

![image](https://user-images.githubusercontent.com/6984103/86530367-d0a01c00-beea-11ea-8553-8006a03740a7.png)

**Comment by jimmy3dita** [comment]
Thanks for crossposting. Is the question the one at this URL? https://stackoverflow.com/questions/62420978/legder-nano-s-cant-effective-authorization

**Comment by wjdfx** [comment]
> Thanks for crossposting. Is this the link to StackOverflow? https://stackoverflow.com/questions/62420978/legder-nano-s-cant-effective-authorization

Yes. I think that after I asked the question, the developer solved the problem and fixed.
